resources_portal/models/material.py

```diff
@@ -10,13 +10,23 @@ class Meta:
         get_latest_by = "created_at"
 
     objects = models.Manager()
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
 
+    material_type = models.CharField(max_length=32)
     url = models.TextField(blank=True)
     pubmed_id = models.CharField(max_length=32, blank=True)
+    url = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
You have this defined twice (also on line 17), and I think I like the other one more. Let's use a TextField that can be blank and can be null.